src/App/pages/MainPage/components/Cards/Cards.tsx

```diff
@@ -30,16 +33,21 @@ const Cards: React.FC<CardsProps> = ({ recipes, handleScroll, onClick }) => {
       loader={<WithLoader loading={loading}>⠀</WithLoader>}
     >
       <div className={styles.main__content}>
-        {recipes?.map((recipe) => (
-          <Card
-            key={recipe.id}
-            image={recipe.image}
-            title={recipe.title}
-            subtitle={recipe.allIngredientsLine}
-            kcal={recipe.calories}
-            onClick={() => onClick(recipe.id)}
-          />
-        ))}
+        {rootStore.recipes.meta === Meta.loading &&
+          rootStore.recipes.recipes.length === 0 &&
+          new Array(BASECOUNT).fill(null).map(() => <CardSkeleton />)}
+        {(rootStore.recipes.meta === Meta.success ||
+          rootStore.recipes.recipes.length > 0) &&
+          recipes?.map((recipe) => (
+            <Card
+              key={recipe.id}
+              image={recipe.image}
+              title={recipe.title}
+              subtitle={recipe.allIngredientsLine}
+              kcal={recipe.calories}
+              onClick={() => onClick(recipe.id)}
```

Comment:
ну конечно было бы хорошо, чтобы `Card` внутри себя в onClick клал свой id и тогда здесь можно бы было написать:
```suggestion
              onClick={onClick}
```

а так сейчас заново создаётся новый колбэк и `Card` не мемоизируется